From: Jan Engelhardt <jengelh@inai.de>
Date: 2018-01-02 19:17:14.364202580 +0100

Some bundled code (which is also an older version and cannot simply be switched
out just yet) needs to be explicitly set to STATIC, or cmake will generate
files like "liblzma.so" etc. with no versioning and which are not installed by
cmake_install.

---
 CMakeLists.txt       |    1 -
 dumb/CMakeLists.txt  |    2 +-
 gdtoa/CMakeLists.txt |    2 +-
 lzma/CMakeLists.txt  |    2 +-
 4 files changed, 3 insertions(+), 4 deletions(-)

Index: gzdoom-g3.2.4/dumb/CMakeLists.txt
===================================================================
--- gzdoom-g3.2.4.orig/dumb/CMakeLists.txt
+++ gzdoom-g3.2.4/dumb/CMakeLists.txt
@@ -24,7 +24,7 @@ endif()
 
 include_directories( include )
 
-add_library( dumb
+add_library( dumb STATIC
     src/core/unload.c
     src/core/rendsig.c
     src/core/rendduh.c
Index: gzdoom-g3.2.4/gdtoa/CMakeLists.txt
===================================================================
--- gzdoom-g3.2.4.orig/gdtoa/CMakeLists.txt
+++ gzdoom-g3.2.4/gdtoa/CMakeLists.txt
@@ -35,7 +35,7 @@ if( NOT MSVC AND NOT APPLE )
 	set( GEN_FP_DEPS ${CMAKE_CURRENT_BINARY_DIR}/arith.h ${CMAKE_CURRENT_BINARY_DIR}/gd_qnan.h )
 endif()
 
-add_library( gdtoa
+add_library( gdtoa STATIC
 	${GEN_FP_FILES}
 	dmisc.c
 	dtoa.c
Index: gzdoom-g3.2.4/lzma/CMakeLists.txt
===================================================================
--- gzdoom-g3.2.4.orig/lzma/CMakeLists.txt
+++ gzdoom-g3.2.4/lzma/CMakeLists.txt
@@ -34,5 +34,5 @@ else()
 	set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_7ZIP_ST" )
 endif()
 
-add_library( lzma ${LZMA_FILES} )
+add_library( lzma STATIC ${LZMA_FILES} )
 target_link_libraries( lzma )
